### ISSUE TITLE
Cosmetic bug when increasing or decreasing bag size

### ISSUE THREAD

**Issue by Xaero252** [obble/modui_classic#53]
When equipping a larger or smaller bag the bag frame doesn't update.
RegisterEvent("BAG_CLOSED") and calling the bag skin function again should solve this.
Will try and test next chance I get.

**Comment by obble** [comment]
Yeah this has been a long-standing bug - I think the logic you suggest may already be happening, I need to take a look at it again

**Comment by Xaero252** [comment]
I looked through the repo and there is no inclusion of the `BAG_CLOSED` event anywhere. This event is fired any time a bag is (re)moved; the `BAG_OPEN` event is fired any time a bag is equipped in a slot. Would need to hook both of these functions to get proper cosmetic behavior, I believe. As equipping into an empty slot will not fire `BAG_CLOSED` but will fire BAG_OPEN` and moving bags between slots will not fire `BAG_OPEN` but will fire `BAG_CLOSED`. I'm not certain replacing a bag will fire `BAG_OPEN`, either, though it may - I'd have to test.
